packages/manager/modules/pci/src/projects/project/storages/databases/database/metrics/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "pci_database_metrics_title": "Metrics",
+  "pci_databases_metrics_title": "Metrics",
+  "pci_databases_metrics_refresh_label": "Auto-Refresh (1M)",
+  "pci_databases_metrics_range_label": "Time range",
+  "pci_databases_metrics_range_label_1H": "1H",
+  "pci_databases_metrics_range_label_1D": "1D",
+  "pci_databases_metrics_range_label_1W": "1W",
+  "pci_databases_metrics_range_label_1M": "1M",
+  "pci_databases_metrics_range_label_1Y": "1Y",
```

Comment:
No need for translation if we display raw value

Reply:
That's what I originally thought but I felt like it was better to "future proof" it. Even in french, it feels weird to have 1D for "1 jour".